app/cisc/commands.go

```diff
@@ -0,0 +1,193 @@
+				Action:  configUpdate,
+			},
+			{
+				Name:    "list",
+				Aliases: []string{"ls"},
+				Usage:   "list existing config and proposed",
+				Action:  configList,
+				Flags: []cli.Flag{
+					cli.BoolFlag{
+						Name:  "p,propose",
+						Usage: "will also show proposed config",
+					},
+				},
+			},
+			{
+				Name:    "vote",
+				Aliases: []string{"v"},
+				Usage:   "vote on existing config",
+				Flags: []cli.Flag{
+					cli.BoolFlag{
+						Name:  "r,reject",
+						Usage: "reject the proposition",
+					},
+				},
+				Action: configVote,
+			},
+		},
+	}
+	commandKeyvalue = cli.Command{
+		Name:    "keyvalue",
+		Aliases: []string{"kv"},
+		Usage:   "storing and retrieving key/value pairs",
+		Subcommands: []cli.Command{
+			{
+				Name:    "list",
+				Aliases: []string{"ls"},
+				Usage:   "list all values",
+				Action:  kvList,
+			},
+			{
+				Name:      "value",
+				Aliases:   []string{"v"},
+				Usage:     "return the value of a key",
+				ArgsUsage: "key",
+				Action:    kvValue,
+			},
+			{
+				Name:      "add",
+				Aliases:   []string{"a"},
+				Usage:     "add a new key/value pair",
+				ArgsUsage: "key value",
+				Action:    kvAdd,
+			},
+			{
+				Name:      "rm",
+				Aliases:   []string{"ls"},
+				Usage:     "list all values",
+				ArgsUsage: "key",
+				Action:    kvRm,
+			},
+		},
+	}
+	commandSSH = cli.Command{
+		Name:  "ssh",
+		Usage: "handling your ssh-keys",
+		Subcommands: []cli.Command{
+			{
+				Name:    "add",
+				Aliases: []string{"a"},
+				Usage:   "adds a new entry to the config",
+				Action:  sshAdd,
+				Flags: []cli.Flag{
+					cli.StringFlag{
+						Name:  "a,alias",
+						Usage: "alias to use for that entry",
+					},
+					cli.StringFlag{
+						Name:  "u,user",
+						Usage: "user for that connection",
+					},
+					cli.StringFlag{
+						Name:  "p,port",
+						Usage: "port for the connection",
+					},
+				},
+			},
+			{
+				Name:      "del",
+				Aliases:   []string{"rm"},
+				Usage:     "deletes an entry from the config",
+				ArgsUsage: "alias_or_host",
+				Action:    sshDel,
+			},
+			{
+				Name:    "list",
+				Aliases: []string{"ls"},
+				Usage:   "shows all entries for this device",
+				Action:  sshLs,
+				Flags: []cli.Flag{
+					cli.BoolFlag{
+						Name:  "a,all",
+						Usage: "show entries for all devices",
+					},
+				},
+			},
+			//{
+			//	Name:    "rotate",
+			//	Aliases: []string{"r"},
+			//	Usage:   "renews all keys - only active once the vote passed",
+			//	Action:  sshRotate,
+			//},
+			//{
+			//	Name:    "sync",
+			//	Aliases: []string{"tc"},
+			//	Usage:   "sync config and blockchain - interactive",
+			//	Flags: []cli.Flag{
+			//		cli.StringFlag{
+			//			Name:  "tob,toblockchain",
+			//			Usage: "force copy of config-file to blockchain",
+			//		},
+			//		cli.StringFlag{
+			//			Name:  "toc,toconfig",
+			//			Usage: "force copy of blockchain to config-file",
+			//		},
+			//	},
+			//	Action: sshSync,
+			//},
```

Comment:
Usually I'd say yes, but since he already wrote them, and the commands are already there (but with a `log.Fatal("Not implemented")`, I'd keep them here since it's likely to be implemented in the near future...